clang/test/dpct/cmake_migration/case_034/input.cmake

```diff
@@ -0,0 +1,23 @@
+target_compile_features(foo PUBLIC cuda_std_03)
+
+target_compile_features(foo PUBLIC cuda_std_14)
+
+target_compile_features(foo PUBLIC cuda_std_26)
```

Comment:
Pls add comments for line5, like:
# In the following cases no change is expected but due to limitations of
# pattern-rewriter the C++ standard is updated to C++17. This will be fixed in
# future.

Reply:
Done.